Makefile

```diff
@@ -85,6 +85,20 @@ build-esp32-bin:
 		--merge \
 		target/xtensa-esp32-espidf/release/micro-rdk-server-esp32.bin
 
+build-esp32c6-bin:
+	cargo +nightly espflash save-image \
```

Comment:
Note: this decouples us from a pinned rust version for using the non-espressif rust toolchain for riscv compiles. I'm not sure I love that, open to suggestions. We could also use the `+esp` toolchain for this, but I was sort of enthusiastic about leaving that behind.

Reply:
do we have to use nightly channel here?

Reply:
Yes, per https://docs.esp-rs.org/book/installation/riscv.html, for `std`, we need the nightly channel.